src/main.cpp

```diff
@@ -14,4 +15,13 @@ int main() {
     // Literal syntax
     auto a1 = "a1"_sq;
     uint64_t board = 1ULL << static_cast<int>(a1);
+
+    Bitboard myboard;
+    std::cout << myboard << std::endl;
+
+    auto d7 = "d7"_sq;
```

Comment:
Is this line necessary?

Reply:
you have marked 4 lines oldman

Reply:
My bad. I meant the `auto d7 = "d7"_sq;` line.

Reply:
It is not necessary.